package_drupal_script.sh

```diff
@@ -135,8 +135,13 @@ git clone --config core.autocrlf=false --config core.eol=lf --quiet https://git.
 pushd ${STAGING_DIR}/sprint/drupal8 >/dev/null
 cp ${REPO_DIR}/example.gitignore ${STAGING_DIR}/sprint/drupal8/.gitignore
 
-echo "Running composer install --quiet"
-composer install --quiet
+echo "Running ddev composer install --quiet"
+set -x
+ddev config --project-type=drupal8
+ddev start
+ddev composer install --quiet
```

Comment:
Will this still be available locally via the docker volume (can't remember if it's a bind mount) or the bgsync option? If not, it might lead to some unexpected issues if looking at code in a local editor.

Reply:
This is the package script, so it's running only on the testbot, and will never be available to a user. The purpose here is to use `ddev composer` instead of host-installed composer to create the initial version of the vendor directory. 

I'm not sure it was a good idea, but because composer 1.9.0 is required for core dev, and since of course it wasn't up to that level on the testbots (but is in the web container) I thought this problem could be eliminated forever by just using `ddev composer`. But the current test failure on Windows may talk me out of that. Waiting to see!

Reply:
Oh, forgot to mention, bgsync is long-deprecated (and worked only on macOS). But anyway, this is all about code that gets created, which is bind-mounted from the host.